ota-plus-web/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.17
```

Comment:
Why don't you bump it to the latest?

Reply:
You mean 1.x? Some services seem to be using 0.13.x while others use 1.x. Don't know what's the story behind that and if it's safe to bump all the services to 1.x, maybe @koshelev @simao?

Reply:
I just bumped some of them while working on a service. So feel free to make the change

Reply:
Discussed offline. Some sbt plugins were not available for 1.x, maybe they are now.

Reply:
I had to make more changes than I expected to move to sbt 1.2.8, so I thought I'd better open a new PR: #1967.